kernel/src/kernel_tests.rs

```diff
@@ -60,7 +60,7 @@ impl TestContext {
         let device_tree = crate::device_tree::DeviceTree::new(device_tree_address);
         let mut pmm = mm::PhysicalMemoryManager::from_device_tree(
             &device_tree,
-            crate::MemoryImpl::get_page_size(),
+            crate::PagingImpl::get_page_size(),
```

Comment:
```suggestion
            PagingImpl::get_page_size(),
```
Since there's a `use` statement above?

Reply:
If I remember correctly `use crate::paging::PagingImpl` brings in the PagingImpl trait whereas
`crate::PagingImpl` is the actual type implementing `crate::paging::PagingImpl`

Does this pass the type-checker ?

Reply:
No, closing